redisstats/handler_test.go

```diff
@@ -0,0 +1,83 @@
+package redisstats
+
+import (
+	"errors"
+	"testing"
+
+	"net"
+
+	"github.com/segmentio/redis-go"
+	"github.com/segmentio/stats"
+)
+
+func TestHandler(t *testing.T) {
+	metricHandler := &testMetricHandler{}
+	e := stats.NewEngine("")
+	e.Register(metricHandler)
+
+	wrappedRedisHandler := NewHandlerWith(e, &testRedisHandler{})
```

Comment:
The variable name here is quite verbose, Go guidelines usually recommend using short names as long as it doesn't impact readability. For example here we're already in the redisstats package so repeating that it's a redis handler is kind of redundant.

That being said, naming is not an exact science so feel free to leave it as it is if you feel like it helps better communicate what the code is about.